.github/workflows/liveness-branch-sync.yml

```diff
@@ -0,0 +1,22 @@
+name: Liveness Release Branch Sync
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
```

Comment:
We might have to add a pemission block like

```
  permissions:
    pull-requests: write
``` 

because Amplify UI repo is configured to only grant readonly access to `GITHUB_TOKEN` (used below) unless otherwise specified.